README.md

```diff
@@ -1,13 +1,13 @@
-# Template for a ZetaChain Hardhat Project
+# Hardhat Template for ZetaChain
 
 This is a simple Hardhat template that provides a starting point for developing
-smart contract applications on the ZetaChain blockchain.
+smart contract applications on ZetaChain.
 
 ## Prerequisites
 
 Before getting started, ensure that you have
-[Node.js](https://nodejs.org/en/download) and [Yarn](https://yarnpkg.com/)
-installed on your system.
+[Node.js](https://nodejs.org/en/download) (version 18 or above) and
```

Comment:
do you think the final `and` is necessary? as is a new line looks like an enum list

Reply:
then is fine, I thought was render in different lines